dod/src/main/java/com/revolut/rxdata/dod/DataObservableDelegate.kt

```diff
@@ -98,23 +98,24 @@ class DataObservableDelegate<Params : Any, Domain : Any> constructor(
      * Requests data from network and subscribes to updates
      * (can be triggered by other subscribers or manual cache overrides)
      *
-     * @param forceReload - if true network request will be made even if data exists in caches
+     * @param loadingStrategy - if [LoadingStrategy.ForceReload] - network request will be made even if data exists in caches,
```

Comment:
there are two places with strategy docs - here and in LoadingStrategy class itself. Let's keep it in one place - in LoadingStrategy file

Reply:
done